.changeset/heavy-points-marry.md

```diff
@@ -0,0 +1,43 @@
+---
+'@primer/react': major
+---
+
+![NewLabels](https://user-images.githubusercontent.com/2313998/155384037-8a56c943-4cff-4349-925b-dc28247e3491.png)
```

Comment:
OOO shiny ✨ ... We should do this for the other components too if we're doing this here. Thoughts @mperrotti?

Reply:
You mean show all the variations? That might not fit neatly into 1 header image 😅

Reply:
Sorry, no I meant other components being deprecated in v35 don't have header images in their release notes, so this will stand out. We should either do it for all, or not at all IMO.